src/index.html

```diff
@@ -8,6 +8,7 @@
 	<meta name="description" content="The home of dojo. Documentation. Examples. Community.">
 	<link rel="icon" href="./assets/favicon.ico" />
 	<link rel="stylesheet" type="text/css" href="./assets/prism.css" />
+	<link href="https://use.fontawesome.com/releases/v5.7.0/css/svg-with-js.css" rel="stylesheet"></link>
```

Comment:
Does this work with BTR? I.e. I still don't think you'll be able to get the font awesome classes. Would it be better to import this into `main.css` so the fa classes can be inlined?

Reply:
Yeah. That part is definitely still WIP. I am going to play around with it next OSS day and determine the best option for this.

Reply:
I did end up removing this and going with the manual import as I was able to resolve the bundle size issue.